pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -719,15 +689,17 @@ func TestEtcdUpdateStatus(t *testing.T) {
 	expected.Labels = podIn.Labels
 	expected.Status = podIn.Status
 
-	_, _, err := statusStorage.Update(ctx, &podIn)
+	_, _, err = statusStorage.Update(ctx, &podIn)
 	if err != nil {
 		t.Fatalf("Unexpected error: %v", err)
 	}
-	podOut, err := storage.Get(ctx, "foo")
+	obj, err := storage.Get(ctx, "foo")
 	if err != nil {
 		t.Errorf("unexpected error: %v", err)
 	}
-	if !api.Semantic.DeepEqual(&expected, podOut) {
-		t.Errorf("unexpected object: %s", util.ObjectDiff(&expected, podOut))
+	podOut := obj.(*api.Pod)
+	// can not do full comparison b/c we are adding all the meta fields now
```

Comment:
And here:
Can you please explicitly say that validChangedPod() is changing only Labels, so we just compare this?